vidpub/info.py

```diff
@@ -56,7 +56,7 @@ def __repr__(self):
 
     @_escape_return_string
     def render_video_title(self) -> str:
-        suffix = f" – {self.conference.name}"
+        suffix = f" - {self.conference.name}"
```

Comment:
What character is this changed to? Where is this used? I checked YouTube and we’re using `|` everywhere now.

Reply:
<img width="557" alt="Screenshot 2022-12-20 at 3 54 42 PM" src="https://user-images.githubusercontent.com/25934612/208613260-a88b67bd-a5f2-4dc7-9871-2a4f9bbf79a8.png">
I changed this character because it is suggested to change so by my vscode linter. Probably no need to change.

I haven't noticed the Youtube characters now, just trying to get rid of the annoying warnings from vscode. 
<img width="388" alt="Screenshot 2022-12-20 at 3 56 57 PM" src="https://user-images.githubusercontent.com/25934612/208613675-a1b5eb81-fe26-44cd-ac29-a11b37f36b7a.png">
shows yellow highlight here

Reply:
Use `\u2013` instead

Reply:
##ok~ it's done in c17859e73c43f4cc19d3a14662cdb8bab730740a